.zuul.yaml

```diff
@@ -31,6 +31,16 @@
       # Pass vars to crc cli https://review.rdoproject.org/cgit/config/tree/playbooks/crc/simple-start.yaml#n30
       crc_parameters: '--memory 16000 --disk-size 80 --cpus 6'  # Increase from 14336
 
+- job:
+    name: pvs_crc
+    parent: base-simple-crc
+    description: |
+      Create Pvs in crc Cluster
+    run:
+      - ci/define_pvs.yml
+    vars:
+      scenario: "nightly_bundles"
```

Comment:
FYI, this won't do anything here, since ``scenario`` is a parameter for the ci/prepare.yml, ci/deploy_stf.yml, ci/test_stf.yml and ci/post-collect_logs.yml playbooks.

You will need these lines if you use stf-base as the parent job.